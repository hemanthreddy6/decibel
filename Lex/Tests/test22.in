do_this <- (num1: int): string => ("abc");