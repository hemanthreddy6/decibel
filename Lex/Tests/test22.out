IDENTIFIER
<-
(
IDENTIFIER
:
INT
)
:
STRING
=>
(
STRING_LITERAL
)
;
